sequence_transform.c

```diff
@@ -67,13 +67,22 @@ void sequence_transform_task(void) {
 
 //////////////////////////////////////////////////////////////////
 // Trie key stack
-static uint16_t trie_key_stack_data[SEQUENCE_MAX_LENGTH] = {0};
+static uint16_t trie_key_stack_data[MAX(SEQUENCE_MAX_LENGTH, MAX_BACKSPACES)] = {0};
```

Comment:
this will always be `SEQUENCE_MAX_LENGTH`, no?

Reply:
No, there might even be coherent subsets of my own ruleset that have a longer MAX_BACKSPACE. Here is an example that would break without the MAX here
```
^s@ -> something
^s@u -> should
```
The `max_backspace` is 8, but teh max sequence is only 4